### ISSUE TITLE
Add more of the available info to FrankConfig.xsd and/or refer to the website

### ISSUE THREAD

**Issue by jacodg** [ibissource/frank-doc#88]
For example https://frankdoc.frankframework.org/#!/Pipes/BytesOutputPipe contains a lot of information about the pipe but using code completion you will only see: Output bytes as specified by the input XML.

It would be nice to see more information. In the case of BytesOutputPipe it would probably be to much information to add all Javadoc that is available on the class but at least a certain amount of characters would be nice and a pointer that more information is available at [BytesOutputPipe](https://frankdoc.frankframework.org/#!/Pipes/BytesOutputPipe). Adding the link so the user can click on it would be ideal but if not possible at least show something like ... more information available at the Frank!Doc website.

**Comment by gvanbrakel** [comment]
Currently, the choice is made to limit the classdoc in the XSD to the first sentence, to avoid the XSD to explode.
We could experiment with adding full classdoc.

**Comment by jacodg** [comment]
Yes please do so. And in case we see the XSD explode to a unacceptable size please add something like "... more information available at the Frank!Doc website".
